### ISSUE TITLE
Show number of tests taken per capita in a region

### ISSUE THREAD

**Issue by githrik** [koroonakaart/koroonakaart#41]
Visualized as histogram. Linegraph showing the progression of test taken and positive tests ratio per region.

**Comment by kunstnik** [comment]
Maybe stay on the tests/cases per 10 000, to stay comparable?

**Comment by githrik** [comment]
Yes, of course. The main goal is to find the ratio, that verifies regional positive test results and to find out possible patterns that occur when total tests taken per day in each region is decreasing or rising.

**Comment by keeganmcbride** [comment]
Will take a look at adding this in the next couple of days. Thanks for the suggestion.
